src/System.Globalization.Extensions/src/System.Globalization.Extensions.csproj

```diff
@@ -18,49 +18,17 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
Please remove the default Configuration groups above. There is no longer a Windows config.